script.module.xbmcswift2/addon.xml

```diff
@@ -1,10 +1,13 @@
-<addon id="script.module.xbmcswift2" name="xbmcswift2" provider-name="Jonathan Beluch (jbel)" version="2.4.0">
+<addon id="script.module.xbmcswift2" name="xbmcswift2" provider-name="Jonathan Beluch (jbel), Team Kodi" version="2.5.0">
```

Comment:
If you add Team Kodi as provider, Team Kodi should have access to the upstream repo.

Reply:
There are others such as library.node editor that also follow the same logic: https://github.com/XBMC-Addons/plugin.library.node.editor/blob/master/addon.xml#L2

I moved the addon to https://github.com/XBMC-Addons isn't that enough?

Reply:
I know that there are others, but Team Kodis GitHub organisation is https://github.com/xbmc.

Reply:
changed to enen92.